### DISCUSSION TITLE
Search and Switch Browser Tabs in Recently Used Order

### DISCUSSION THREAD

**Discussion by MartinLichtblau** [Flow-Launcher/Flow.Launcher#1184] [Ideas]
Feature Request: The ability to search through browser tabs that are currently open and switch between them in recently used order. Furthermore the ability to switch between any open app in the same way, to quickly cycle through all open views through a single interface. 
Example: switching back and forth between a Stack Overflow tab, github repo and IntelliJ code, no matter on which Desktop or monitor the app/tab is.

Do you plan to implement such a feature? 
What do you think of it?

**Comment by Garulf** [comment]
I remember using a similar plugin in Alfred for Mac using Safari. Not sure if there is any API for that on the windows side.

**Comment by jjw24** [comment]
This would be a good one to have for WindowWalker plugin. Like @Garulf mentioned, not sure of the API for it yet.
